nautobot/core/settings.yaml

```
@@ -0,0 +1,1174 @@
+              execution_time: 3.474244
+            - name: >-
+                  test_run_now_constrained_permissions
+                  (nautobot.extras.tests.test_views.JobTestCase)
+              execution_time: 2.727531
+        ```
+    "environment_variable": "TEST_PERFORMANCE_BASELINE_FILE"
+    "type": "string"
+    "version_added": "1.5.0"
+  "TEST_USE_FACTORIES":
+    "default": false
+    "description": "If set to `True`, the Nautobot test runner will call `nautobot-server generate_test_data ...` before executing any test cases, pre-populating the test database with various pseudo-random instances of many of Nautobot's data models."
+    "details": |
+        !!! warning
+            This functionality requires the installation of the [`factory-boy`](https://pypi.org/project/factory-boy/) Python package, which is present in Nautobot's own development environment, but is _not_ an inherent dependency of the Nautobot package when installed otherwise, such as into a plugin's development environment.
+
+        !!! info
+            Setting this to `True` is a requirement for all Nautobot core tests as of 1.5.0, and it is set accordingly in `nautobot/core/tests/nautobot_config.py`, but defaults to `False` otherwise so as to remain backwards-compatible with plugins that also may use the Nautobot test runner in their own test environments, but have not yet updated their tests to account for the presence of this test data.
+
+        Because this test data can obviate the need to manually construct complex test data, and the random factor can improve test robustness, plugin developers are encouraged to set this to `True` in their configuration, ensure that their development environments include the `factory-boy` Python package as a test dependency, and update their tests as needed.
+    "environment_variable": "NAUTOBOT_TEST_USE_FACTORIES"
+    "type": "boolean"
+    "version_added": "1.5.0"
+  "TIME_FORMAT":
+    "default": "g:i a"
+    "description": "Custom format for times. The default results in strings like \"1:23 p.m.\"."
+    "details": "For detailed instructions on writing format strings, please see [the Django documentation](https://docs.djangoproject.com/en/stable/ref/templates/builtins/#date)."
+    "environment_variable": "NAUTOBOT_TIME_FORMAT"
+    "type": "string"
+  "TIME_ZONE":
+    "default": "UTC"
+    "type": "string"
+  "UI_RACK_VIEW_TRUNCATE_FUNCTION":
+    "$ref": "#/definitions/callable"
+    "default": "UI_RACK_VIEW_TRUNCATE_FUNCTION"
+    "description": "This setting function is used to perform the rack elevation truncation feature."
+    "details": "This provides a way to tailor the truncation behavior to best suit the needs of the installation. The function must take only one argument: the device display name, as a string, attempting to be rendered on the rack elevation. The function must return only one argument: a string of the truncated device display name."
+    "version_added": "1.4.0"
+"type": "object"
```

Just a suggestion. I never see this actually used
```suggestion
"type": "object"
...
```